profiles/targets/desktop/package.use

```diff
@@ -1,6 +1,10 @@
 # Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
+# Andreas Sturmlechner <asturm@gentoo.org> (2024-03-07)
+# Common flags needed by Qt6 revdeps
+dev-qt/qttools designer qdbus
```

Comment:
Also fwiw I wouldn't mind flipping qdbus by default in the ebuild rather than profile-specific, it's kind of a trivial one.

Reply:
if you want to I'm not stopping you. Revdeps are: kinfocenter, plasma-workspace, konversation.

Reply:
Done in commit 00fab6132aa19dba0e3d0640c2a498dc671a28eb